utils/clientProcessing.js

```diff
@@ -0,0 +1,76 @@
+
+/**
+ * Comparator for sorting clients based on their email ids.
+ * 
+ * @param {*} a 
+ * @param {*} b 
+ */
+function compareEmails(a, b) {
+    if (a.email < b.email){
+        return -1;
+    }
+    else if (b.email < a.email){
+        return 1;
+    }
+    return 0;
+}
+
+/**
+ * Given an array of clients from the MindBody API and an array of clients
+ * in the database associated with a specific job, two arrays are returned. The first
+ * being clients from API but not in the database and the second being clients in
+ * the database but not from API.
+ * 
+ * @param {*} clients array of clients from api call, sorted by email.
+ * @param {*} storedClients array of clients from database, sorted by email.
+ * @param {*} jobId job_id of the class the clients are attending.
+ */
+async function processClients(clients, storedClients, jobId) {
```

Comment:
This is an amazing function, but kinda wordy and long, try using some of the javascript array functions like filter, etc. If you need any help we can talk about it in the coding session today